### PR TITLE
[Security] IoTSecuritySolution, IoTSecuritySolutionAnalytics and DeviceSecurityGroups

### PR DESCRIPTION
**Pull request by isra-fel**
## Description

(Cherry-picked commits in #11352 to create this PR. Original author: @MichalHel )

Added new API for the following services: IoTSecuritySolution, IoTSecuritySolutionAnalytics and DeviceSecurityGroups

Design: Azure/azure-powershell-cmdlet-review-pr#520

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by MichalHel** [comment]
I Have some more fixes to make and also check the Cherry-pick that was done , please don't review until I will update differently.

**Comment by MichalHel** [comment]
I pushed all the updates and now the PR is ready for review.

**Comment by isra-fel** [comment]
As we talked offline, there are some changes to be made to the help files.

File: src/Security/Security/help/Az.Security.md [hunk 1]
```diff
@@ -1,91 +1,142 @@
...
+Module Guid: 2b6bac56-a03a-4606-8b05-c1a1f36248db
+Download Help Link: {{ Update Download Link }}
+Help Version: {{ Please enter version of help manually (X.X.X.X) format }}
 Locale: en-US
 ---
 
 # Az.Security Module
 ## Description
-Azure Security Center gives you control over the security of your Azure subscriptions and other machines that you connected to it outside of Azure.
+{{ Fill in the Description }}
```
Comment (isra-fel) [inline src/Security/Security/help/Az.Security.md:11]: There's something wrong in this file -- contents were replaced by placeholders.
I recommend:
1. Replace this file with the version on master https://github.com/Azure/azure-powershell/blob/master/src/Security/Security/help/Az.Security.md
1. Follow this guide to update markdowns https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module

**Comment by MichalHel** [comment]
Fixed

**Comment by isra-fel** [comment]
LGTM

**Pull request merged**